src/node_contextify.cc

```diff
@@ -184,6 +184,32 @@ Local<Context> ContextifyContext::CreateV8Context(
   CHECK(name->IsString());
   Utf8Value name_val(env->isolate(), name);
 
+  Local<Value> codegen = options_obj->Get(env->context(),
+      FIXED_ONE_BYTE_STRING(env->isolate(), "codeGeneration"))
+    .ToLocalChecked();
```

Comment:
Can you use 4 spaces indentation for these lines as well, like other statement continuations?